src/components/language/query.helpers.ts

```diff
@@ -0,0 +1,33 @@
+import { inArray, Query } from 'cypher-query-builder';
+import { propMatch } from '../project';
+import { LanguageFilters } from './dto';
+
+export function languageListFilter(query: Query, filter: LanguageFilters) {
+  query
+    .call((q) =>
+      filter.sensitivity
+        ? q
+            .match(propMatch('sensitivity'))
+            .where({ sensitivity: { value: inArray(filter.sensitivity) } })
+        : q
+    )
+    .call((q) =>
+      filter.leastOfThese
+        ? q
+            .match(propMatch('leastOfThese'))
+            .where({ leastOfThese: { value: true } })
+        : q
+    )
+    .call((q) =>
+      filter.isSignLanguage
+        ? q
+            .match(propMatch('isSignLanguage'))
+            .where({ isSignLanguage: { value: true } })
+        : q
+    )
+    .call((q) =>
+      filter.isDialect
+        ? q.match(propMatch('isDialect')).where({ isDialect: { value: true } })
+        : q
+    );
```

Comment:
Can't these be merged into a single where clause that would be more performant?

Reply:
I'm not sure, I'm not that familiar with the where clause.  What's the query you're imagining?  I can't think of one that allows for us to match on multiple of these filters together.

Reply:
```ts
.where({
  a: ...,
  b: ...,
})
```
aka
```sql
WHERE a AND b
```

Reply:
Ok I think I got it to work, pushed on the latest commit